.gitignore

```diff
@@ -1,4 +1,5 @@
 .vscode
 .idea
 .envrc
-.direnv
\ No newline at end of file
+.direnv
+.vault_pass
```

Comment:
Sorry I didn't get to this @Audiosutras. This is not the .gitignore that gets put into the project. This is the gitignore for when a dev is working on jade-truffle.